lib/src/client/http2_connection.dart

```diff
@@ -275,3 +273,5 @@ class Http2ClientConnection implements connection.ClientConnection {
     return validator(certificate, authority);
   }
 }
+
+class _ShutdownException implements Exception {}
```

Comment:
This could could potentially be reasonable as an Error - from what it looks like to me it's generally indicating a programming bug?